inginious/frontend/templates/course_admin/task_list.html

```diff
@@ -140,4 +151,35 @@ <h3>$:_("Errors while loading tasks")</h3>
     }
     $$('.sorted_table').sortable($$(this).hasClass("active") ? "enable" : "disable");
 });
+
+var active= false;
+$$("#actionControlSelect").change(function () {
+   if(this.value !== "def"){
+       active=true;
+       $$("#action_task").val(this.value);
+       $$("#message").html("<p>Please select task(s) and then click update button.</p>");
+   }else{
+       active=false;
+       $$(".sorted_table tr").each(function(){
+           $$(this).removeClass("table-active");
+       });
+       $$("#message").html("");
+       $$("#action_task").val("");
+   }
+});
+
+
+$$(".sorted_table tr").click(function (e) {
```

Comment:
Actually this may be cleaner and simplier if the action is performed once the action is selected, not involving the order update mechanism, isn't it ? It can stay as it is, I don't really mind for now.

Reply:
Done! :)